src/spikeinterface/core/channelsaggregationrecording.py

```diff
@@ -142,11 +142,17 @@ def get_traces(
                 # in case channel_indices is slice, it has step 1
                 step = channel_indices.step if channel_indices.step is not None else 1
```

Comment:
I'm confused by this. The if statement says if `channel_indices` is None then we check for a step if it is not None, but it has to be None, no?

Reply:
Never mind I was missing the upper portion of the code. I was looking at the diff wrong: )